lib/tasks/bundlers/generateLibraryPreload.js

```diff
@@ -2,10 +2,24 @@ const log = require("@ui5/logger").getLogger("builder:tasks:bundlers:generateLib
 const moduleBundler = require("../../processors/bundlers/moduleBundler");
 const ReaderCollectionPrioritized = require("@ui5/fs").ReaderCollectionPrioritized;
 
+const DEFAULT_FILE_TYPES = [
+	".js",
+	".control.xml", // XMLComposite
+	".fragment.html",
+	".fragment.json",
+	".fragment.xml",
+	".view.html",
+	".view.json",
+	".view.xml",
+	".properties",
+	".json"
```

Comment:
The general ".json" file type is NOT present in our Maven tooling. I've kept it nevertheless for compatibility with existing projects that use the ui5-tooling already.